pipelines/templates/build.yml

```diff
@@ -9,12 +9,6 @@ parameters:
   PublishArtifactTemplate: /pipelines/template/1es-publish-task.yml@self
 
 steps:
-- task: UseDotNet@2
```

Comment:
You sure we don't need this anymore?

Reply:
Yes, the pipeline successfully ran. It will use what's installed at the pool.